src/fireBase.tsx

```diff
@@ -27,19 +19,19 @@ const firebaseConfig: FireBaseConfig = {
   appId: "1:58190409157:web:4144751306c1297a98afac",
   measurementId: "G-NEFKXBHXXC",
 };
```

Comment:
Do not expose your firebase(project) credentials. Instead store them as variables in `.env` file